pkg/dev-env/docker-compose.yml

```diff
@@ -157,15 +230,15 @@ services:
       POSTGRES_DB: postgres
       POSTGRES_PASSWORD: postgres
 
-  ## Gamedao graph service
+  ## GameDAO graph service
   gamedao-graph-service:
     build: '../service'
     environment:
-      IPFS_API_URL: "https://ipfs.infura.io:5001/api/v0"
-      IPFS_CLIENT_ID: "1v04L2wj5JmI0JgKF5KztV0oN8o"
-      IPFS_CLIENT_SECRET: "85547c6003abb67a6335469d1aa6a3a3"
+      IPFS_API_URL: "${IPFS_API_URL}"
+      IPFS_CLIENT_ID: "${IPFS_CLIENT_ID}"
+      IPFS_CLIENT_SECRET: "${IPFS_CLIENT_SECRET}"
       SERVER_PORT: 4000
-      CHAIN_RPC_URL: "ws://subzero:9944"
+      CHAIN_RPC_URL: "ws://subzero-parachain-a:9944"
```

Comment:
actually we should not use a collator to index, but a full node in archiver mode, right?